docs/Consuming.md

```diff
@@ -97,6 +97,11 @@ Here is an example of topic validation:
 
 ![Topic syntax validation](assets/kafka-file-consumer-topic-syntax-validation.png)
 
+Existing topic validation is done only when cluster is `connected`. It reports error according the broker configuration `auto.create.topics.enable` value.

```

Comment:
If the topic doesn't already exist, an error will be reported if the broker configuration is accessible and `auto.create.topics.enable=false`.

Reply:
fixed